sympy/tensor/tensor.py

```diff
@@ -1539,7 +1526,7 @@ def tensorhead(name, typ, sym=None, comm=0):
     >>> A(a, -b)
     A(a, -b)
 
-    If no symmetry parameter is provided, assume there are not index
+    If no symmetry parameter is provided, assume there are no index
```

Comment:
indices instead of index?

Reply:
Yes, you are right. I am new to this process, so is it better to make the corrections and make a new commit to this pull request or force-push the changes to the existing commit for cleaner history?